saleor/product/models.py

```diff
@@ -423,6 +430,13 @@ class Product(SeoModel, ModelWithMetadata):
         related_name="+",
     )
     rating = models.FloatField(null=True, blank=True)
+    tax_class = models.ForeignKey(
+        TaxClass,
+        related_name="products",
+        blank=True,
+        null=True,
+        on_delete=models.SET_NULL,
```

Comment:
Should we add DEFAULT here?

Reply:
No, in products we actually want to allow `null`. By default we use `product_type.tax_class` and here we can optionally override it.